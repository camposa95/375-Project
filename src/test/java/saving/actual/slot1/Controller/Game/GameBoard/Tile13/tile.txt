TerrainType: PASTURE
DieNumber: 5
HasRobber: false
VertexIDs: [30, 31, 32, 40, 41, 42]
