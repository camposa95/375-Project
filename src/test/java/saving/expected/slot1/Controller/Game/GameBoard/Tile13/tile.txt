TerrainType: MOUNTAINS
DieNumber: 3
HasRobber: false
VertexIDs: [30, 31, 32, 40, 41, 42]
